### DISCUSSION TITLE
load keyword

### DISCUSSION THREAD

**Discussion by slendidev** [odin-lang/Odin#2825] [Ideas/Requests]
This keyword would look something like this:

```
load "my_file"
```

It is like C/C++'s #include, as in it literally copies and pastes the code from another file.

**Comment by Beefster09** [comment]
why would you need this instead of a regular import?

**Reply by slendidev** [reply]
Can be useful in some cases like suckless style configuration

**Comment by Kelimion** [comment]
We already have `#load(filename)` which embeds the file as a `[]u8`, and its optional type param like `#load(filename, string)`.

Either way I'm closing this discussion. For code we have `import`ed, and `#load` covers the embedding use cases. The equivalent of C's `#include` doesn't make sense in Odin.
